src/tests/topp/ConvertTSVToTraML_2_output.TraML

```diff
@@ -81,6 +81,11 @@
       <cvParam cvRef="MS" accession="MS:1001226" name="product ion intensity" value="1"/>
       <cvParam cvRef="MS" accession="MS:1002007" name="target SRM transition"/>
       <userParam name="annotation" type="xsd:string" value="y5"/>
+      <userParam name="detecting_transition" type="xsd:integer" value="1"/>
```

Comment:
do you test for *all* fields, or how do you know the fields are missing? you cannot know until you have seen all `userParam` right?